Sources/MapboxCoreNavigation/BillingHandler.swift

```diff
@@ -7,26 +7,21 @@ import os.log
 @_implementationOnly import MapboxCommon_Private
 
 /// BillingService from MapboxCommon
-private typealias BillingServiceNative = MapboxCommon_Private.BillingService
+private typealias BillingServiceNative = MapboxCommon_Private.BillingServiceFactory
```

Comment:
/cc @S2Ler, there are a couple of billing related changes.

Reply:
Looks good.